circuits/sybil/sybilResCredentialAtomicQuerySigOffChain.circom

```diff
@@ -0,0 +1,256 @@
+
+    for (var i=0; i<GistLevels; i++) { verifyStateSecret.gistMtp[i] <== gistMtp[i]; }
+    verifyStateSecret.gistRoot <== gistRoot;
+    verifyStateSecret.gistMtpAuxHi <== gistMtpAuxHi;
+    verifyStateSecret.gistMtpAuxHv <== gistMtpAuxHv;
+    verifyStateSecret.gistMtpNoAux <== gistMtpNoAux;
+
+    verifyStateSecret.secretClaimHash ==> secretClaimHash;
+    
+    // 3. Compute profile.
+    component selectProfile = SelectProfile();
+    selectProfile.in <== userGenesisID;
+    selectProfile.nonce <== profileNonce;
+    userID <== selectProfile.out;
+
+    // 4. Compute sybil
+    component computeSybilID = ComputeSybilID();
+    computeSybilID.crs <== crs;
+    computeSybilID.stateSecret <== secretClaimHash;
+    computeSybilID.claimHash <== uniClaimHash;
+    sybilID <== computeSybilID.out;
+}
+
+
+template VerifyAndHashUniClaim(IssuerLevels){
+    signal input issuerAuthClaim[8];
+    signal input issuerAuthClaimMtp[IssuerLevels];
+    signal input issuerAuthClaimsRoot;
+    signal input issuerAuthRevRoot;
+    signal input issuerAuthRootsRoot;
+    // signal input issuerAuthState;
+
+    // issuer auth claim non rev proof
+    signal input issuerAuthClaimNonRevMtp[IssuerLevels];
+    signal input issuerAuthClaimNonRevMtpNoAux;
+    signal input issuerAuthClaimNonRevMtpAuxHi;
+    signal input issuerAuthClaimNonRevMtpAuxHv;
+
+    // claim issued by issuer to the user
+    signal input issuerClaim[8];
+    // issuerClaim non rev inputs
+    signal input issuerClaimNonRevMtp[IssuerLevels];
+    signal input issuerClaimNonRevMtpNoAux;
+    signal input issuerClaimNonRevMtpAuxHi;
+    signal input issuerClaimNonRevMtpAuxHv;
+    signal input issuerClaimNonRevClaimsRoot;
+    signal input issuerClaimNonRevRevRoot;
+    signal input issuerClaimNonRevRootsRoot;
+    signal input issuerClaimNonRevState;
+
+    // issuerClaim signature
+    signal input issuerClaimSignatureR8x;
+    signal input issuerClaimSignatureR8y;
+    signal input issuerClaimSignatureS;
+
+    signal input issuerClaimSchema;
+
+    signal input userGenesisID;
+    signal input profileNonce;
+    signal input claimSubjectProfileNonce;
+
+    // inter-signal
+    signal issuerAuthState;
+    // signal uniClaimHash;
+
+    signal output claimHash;
+
+    //  A. Verify issued and not revoked
+    var AUTH_SCHEMA_HASH  = 301485908906857522017021291028488077057;
+    component issuerSchemaCheck = verifyCredentialSchema();
+    for (var i=0; i<8; i++) { issuerSchemaCheck.claim[i] <== issuerAuthClaim[i]; }
+    issuerSchemaCheck.schema <== AUTH_SCHEMA_HASH;
+
+    // verify authClaim issued and not revoked
+    // calculate issuerAuthState
+    component issuerAuthStateComponent = getIdenState();
+    issuerAuthStateComponent.claimsTreeRoot <== issuerAuthClaimsRoot;
+    issuerAuthStateComponent.revTreeRoot <== issuerAuthRevRoot;
+    issuerAuthStateComponent.rootsTreeRoot <== issuerAuthRootsRoot;
+
+    issuerAuthState <== issuerAuthStateComponent.idenState;
```

Comment:
`issuerAuthState` is not connected to anything else down the flow

Reply:
removed